prov/efa/src/rdm/efa_rdm_pke_utils.h

```diff
@@ -58,6 +58,42 @@ size_t efa_rdm_pke_get_segment_offset(struct efa_rdm_pke *pke)
 	return 0;
 }
 
+/**
+ * @brief copy data from the user's buffer to the packet's wiredata.
+ *
+ * @param[in]		iov_mr  memory descriptors of the user's data buffer.
+ * @param[in,out]	pke	    packet entry. Header must have been set when the function is called.
+ * @param[in]		ope		operation entry that has user buffer information.
+ * @param[in]		payload_offset	the data offset in reference to pkt_entry->wiredata.
+ * @param[in]		segment_offset	the data offset in reference to user's buffer
+ * @param[in]		data_size	length of the data to be set up.
+ * @return   		length of data copied.
+ */
+inline size_t
```

Comment:
I missed it earlier: it should be `static inline size_t`

Reply:
Can I call it from efa_rdm_ope.c if it is static?